src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1946,5 +1947,65 @@ public String getVariableNamePattern() {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+		EnumUtils<Horse.Color> horseColorUtils = new EnumUtils<>(Horse.Color.class, "horse colors");
+		Classes.registerClass(new ClassInfo<>(Horse.Color.class, "horsecolor")
+				.user("horse ?colou?rs?")
```

Comment:
Shouldn't it be ``.user("horse ? colo?rs?")`` for consistency?

Reply:
Sorry, I haven't seen any pattern in which there are two spaces and one is optional, and I don't think it should be colo?rs? because some people use colo**u**r

Reply:
The space is my mistake, but if you want to add [u] as an optional character, shouldn't it be something like ```.user("enchant[ment][ ]offers?")```? Rather than ```.user("horse ?colou?rs?")```

Reply:
Both methods work and the one I'm currently using is the most commonly used one, so I'm not sure